src/hub.js

```diff
@@ -574,6 +572,8 @@ async function runBotMode(scene, entryManager) {
 }
 
 document.addEventListener("DOMContentLoaded", async () => {
+  await store.init();
```

Comment:
Are we sure nothing else ends up trying to access store before this runs?

Reply:
Currently, init really just initializes the avatarId in the profile. I'll change the method name to make that more clear

Reply:
There might be other parts of the code that relies on the store before this happens, but afaict, they do not rely on the avatarId specifically